### ISSUE TITLE
Sybase JDBC DB Plugin

### ISSUE THREAD

**Issue by adam133** [kestra-io/plugin-jdbc#281]
### Feature description

I'd like to build a flow to pull some data from a Sybase database. 

I'm currently able to connect using the `jconn4.jar` jdbc driver - I believe you can find a download for it here: https://maven.jumpmind.com/repo/jdbc/sybase/jconn4/16.0/

**Comment by adam133** [comment]
Should be able to use this Docker file to test a Sybase database. 

```
# found here: https://github.com/iambearman/docker-sybase/blob/main/Dockerfile
# updated version from here: https://hub.docker.com/r/datagrip/sybase
# to include the -T11889 option to the RUN_MYSYBASE file
# and to remove the last line of the RUN_MYSYBASE file
# this option is necessary to keep sybase from crashing when virtualized
FROM datagrip/sybase:16.0

RUN sed -i '$ d' /opt/sybase/ASE-16_0/install/RUN_MYSYBASE
RUN printf -- '-T11889 \\\n\n' >> /opt/sybase/ASE-16_0/install/RUN_MYSYBASE

EXPOSE 3001/tcp

# user: tester
# password: guest1234
# database: testdb
```

**Comment by loicmathieu** [comment]
Fixed by #282
